service/web/tabs/app/src/components/StopAllBackfillsButton.tsx

```diff
@@ -46,7 +51,24 @@ class StopAllRealButton extends React.Component<IStopAllBackillsProps, IStopAllB
   }
 }
 
-class StopAllButton extends React.Component {
+interface IStopAllBackillsProps {
+}
+
+interface IStopAllBackfillsState {
+  disabled: boolean
+}
+
+class StopAllButton extends React.Component<IStopAllBackillsProps, IStopAllBackfillsState> {
+  handleTextChange(newValue: String) {
+    this.setState({
+      disabled: newValue != "break glass"
```

Comment:
`I realize that by breaking glass that I am imposing a cost on the whole Block organization.` ?

Reply:
Since this is a public repo though:
`I realize that by breaking glass that I am imposing a cost on the whole organization.`

Reply:
bit too condescending imo

Reply:
Which part do you have issue with?
`break glass and stop ALL backfills` ?

Reply:
yeah sure

Reply:
didn't see @shellderp 's comments originally, I think github was havin a day - happy to make this into whatever folks it to say